docs/develop/03_workshop/04_attester/01_account.md

```diff
@@ -66,14 +66,9 @@ The `generateAccount` method returns an object with the following two properties
 Generating these values takes two steps:
 
 1. Create the `mnemonic` value using the `mnemonicGenerate()` method from the `Utils.Crypto` package.
-2. The `account` value first needs a `keyring` value defined, which is a data structure for defining the key pair type with the following parameters:
+2. The `account` value first needs a `keyring` value defined, which is a data structure for defining the key pair type as `ed25519`.
```

Comment:
Ed25519 is simply one of the 3 key types/signature algorithms supported. We typed everything as `ed25519` because that was easiest (and is the default if you don't change the key type using the `type` parameter on `makeKeypairFromUri`), but strictly speaking any of 'sr25519', 'ed25519', or 'ecdsa' work.